src/framing.rs

```diff
@@ -862,8 +862,13 @@ fn codec() {
 fn padding() {
     use crate::ciphersuite::*;
     use crate::utils::*;
+    use rand::RngCore;
 
-    let ciphersuite = Ciphersuite::new(Name::MLS10_128_DHKEMX25519_CHACHA20POLY1305_SHA256_Ed25519);
+    fn random_usize() -> usize {
```

Comment:
We should have that function in `utils.rs`

Reply:
Nope, it's only used in a test. So if you want it outside it must go into a test_util.

Reply:
All the testing needs quite some clean-up. But I figured this PR is already big enough.